react/components/ProductPrice/index.js

```diff
@@ -286,62 +297,8 @@ ProductPrice.Loader = (loaderProps = {}) => (
 
 ProductPrice.Loader.displayName = 'ProductPrice.Loader'
 
-const priceWithIntl = injectIntl(ProductPrice)
+const ProductPriceWithIntl = injectIntl(ProductPrice)
 
-priceWithIntl.schema = {
-  title: 'admin/editor.productPrice.title',
-  description: 'admin/editor.productPrice.description',
-  type: 'object',
-  properties: {
-    labelSellingPrice: {
-      type: 'string',
-      title: 'admin/editor.productPrice.labelSellingPrice',
-      default: ProductPrice.defaultProps.labelSellingPrice,
-      isLayout: false,
-    },
-    labelListPrice: {
-      type: 'string',
-      title: 'admin/editor.productPrice.labelListPrice',
-      default: ProductPrice.defaultProps.labelListPrice,
-      isLayout: false,
-    },
-    showSellingPriceRange: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showSellingPriceRange',
-      default: ProductPrice.defaultProps.showSellingPriceRange,
-      isLayout: true,
-    },
-    showListPriceRange: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showListPriceRange',
-      default: ProductPrice.defaultProps.showListPriceRange,
-      isLayout: true,
-    },
-    showListPrice: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showListPrice',
-      default: ProductPrice.defaultProps.showListPrice,
-      isLayout: true,
-    },
-    showLabels: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showLabels',
-      default: ProductPrice.defaultProps.showLabels,
-      isLayout: true,
-    },
-    showInstallments: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showInstallments',
-      default: ProductPrice.defaultProps.showInstallments,
-      isLayout: true,
-    },
-    showSavings: {
-      type: 'boolean',
-      title: 'admin/editor.productPrice.showSavings',
-      default: ProductPrice.defaultProps.showSavings,
-      isLayout: true,
-    },
-  },
-}
+ProductPriceWithIntl.defaultProps = ProductPrice.defaultProps
```

Comment:
no need for this

Reply:
This was needed because another component imports the `defaultProps` from this component for its schema.

(I won't get into whether this is a good architecture or not, but I inherited the code this way  and it works for now)